### DISCUSSION TITLE
Audi RSQ3 memorize RS Mode

### DISCUSSION THREAD

**Discussion by jannikuhl** [jilleb/MQB-FPA#89] [Exploration and research]
Hi, has anyone already looked into RS 1 and RS 2 modes of recent Audi models?
I have a 2023 RSQ3 with Gateway 3Q0 907 530 AR, by using the template, I was only able to make it stay in RS 1 Mode over ignition cycles. However, RS 1 always resets.

I attached the original dataset. Maybe someone has an idea?

[19_RSQ3_3Q0907530AR_WUAZZZF30P1902240.zip](https://github.com/jilleb/MQB-FPA/files/11320647/19_RSQ3_3Q0907530AR_WUAZZZF30P1902240.zip)

**Comment by Mr-MIBonk** [comment]
A little question to your car, how Start/Stop works, is it disabled or enabled automatic in some mode?

**Reply by jannikuhl** [reply]
It is enabled in all modes and as soon as the ignition cycled it turns on again if it was manually disabled.

**Reply by jjff2** [reply]
> A little question to your car, how Start/Stop works, is it disabled or enabled automatic in some mode?

hi sir, there was a chart amount_of_unknown38_rows_with_value, but it's hided in the new update. may i know why?

**Reply by jilleb** [reply]
You can unhide it, by uncommenting. I disabled it for now because it speeds up the output when I'm testing some things.

**Reply by jjff2** [reply]
Thanks for reply. it has meaning to the group. I will unhide it.

**Comment by jjff2** [comment]
> Hi, has anyone already looked into RS 1 and RS 2 modes of recent Audi models? I have a 2023 RSQ3 with Gateway 3Q0 907 530 AR, by using the template, I was only able to make it stay in RS 1 Mode over ignition cycles. However, RS 1 always resets.
> 
> I attached the original dataset. Maybe someone has an idea?
> 
> [19_RSQ3_3Q0907530AR_WUAZZZF30P1902240.zip](https://github.com/jilleb/MQB-FPA/files/11320647/19_RSQ3_3Q0907530AR_WUAZZZF30P1902240.zip)

Sorry for ignore you last time i saw the page. Did you edit “ Remember setting in specific mode after restart of the car”?
